apischeme_SSS.py

```diff
@@ -76,20 +100,10 @@ def get_bastion_ips(resource):
 
 ingress = resource.spec.managementAPIServerIngress
 
-if set(ingress.allowedCIDRBlocks) == all_ips:
+ingress_ips = set(ingress.allowedCIDRBlocks)
+if ingress_ips == all_ips:
     print("Same IPs, no-op\n%s" % all_ips)
     sys.exit(0)
 
-# Overwrite the list of IPs
-ingress.allowedCIDRBlocks = list(all_ips)
-print("Applying IPs: %s" % ingress.allowedCIDRBlocks)
-
-# Tell cloud-ingress-operator it's okay to apply the CIDRs now.
-if not ingress.enabled:
-    print("Enabling ingress")
-    ingress.enabled = True  # As opposed to the string "true".
-
-sss_resources = dyn_client.resources.get(
-    api_version="hive.openshift.io/v1", kind="SelectorSyncSet"
-)
-dyn_client.apply(sss_resources, body=sss.to_dict())
+missing_ips = [ip for ip in all_ips if ip not in ingress_ips]
+add_missing_ips(missing_ips)
```

Comment:
Do we need exception handling/retries?

Reply:
i thought about that, but figured we'd want to see the errors in full if they happen.
to the best of my knowledge, boto3 has a built-in retry mechanism, so we don't need to specify retries explicitly.
does that make sense?